dymos/examples/robot_arm/test/test_robot_arm.py

```diff
@@ -0,0 +1,152 @@
+        phase.add_state('x1', fix_initial=True, fix_final=True, rate_source='x1_dot', targets=['x1'], units='rad')
+        phase.add_state('x2', fix_initial=True, fix_final=True, rate_source='x2_dot', targets=['x2'], units='rad')
+        phase.add_state('x3', fix_initial=True, fix_final=True, rate_source='x3_dot', targets=['x3'], units='m/s')
+        phase.add_state('x4', fix_initial=True, fix_final=True, rate_source='x4_dot', targets=['x4'], units='rad/s')
+        phase.add_state('x5', fix_initial=True, fix_final=True, rate_source='x5_dot', targets=['x5'], units='rad/s')
+
+        phase.add_control('u0', opt=True, lower=-1, upper=1, scaler=0.1, targets=['u0'], units='m**2/s**2',
+                          rate_continuity=False)
+        phase.add_control('u1', opt=True, lower=-1, upper=1, scaler=0.1, targets=['u1'], units='m**3*rad/s**2',
+                          rate_continuity=False)
+        phase.add_control('u2', opt=True, lower=-1, upper=1, scaler=0.1, targets=['u2'], units='m**3*rad/s**2',
+                          rate_continuity=False)
+
+        phase.add_objective('time', scaler=1.0)
+
+        phase.set_refine_options(refine=True, tol=1e-6, smoothness_factor=1.2)
+
+        p.setup(check=True, force_alloc_complex=False, mode='auto')
+        p.set_val('traj.phase.t_initial', 0)
+        p.set_val('traj.phase.t_duration', 10)
+        p.set_val('traj.phase.states:x0', phase.interpolate(ys=[4.5, 4.5], nodes='state_input'))
+        p.set_val('traj.phase.states:x1', phase.interpolate(ys=[0.0, 2*np.pi/3], nodes='state_input'))
+        p.set_val('traj.phase.states:x2', phase.interpolate(ys=[np.pi/4, np.pi/4], nodes='state_input'))
+        p.set_val('traj.phase.states:x3', phase.interpolate(ys=[0.0, 0.0], nodes='state_input'))
+        p.set_val('traj.phase.states:x4', phase.interpolate(ys=[0.0, 0.0], nodes='state_input'))
+        p.set_val('traj.phase.states:x5', phase.interpolate(ys=[0.0, 0.0], nodes='state_input'))
+
+        return p
+
+    def test_partials(self):
+        p = self.make_problem(transcription=Radau, optimizer='SLSQP')
+        p.run_model()
+        with printoptions(linewidth=1024, edgeitems=100):
+            cpd = p.check_partials(method='fd', compact_print=True, out_stream=None)
+
+    def test_robot_arm_radau(self):
+        p = self.make_problem(transcription=Radau, optimizer='IPOPT', numseg=30)
+        dm.run_problem(p, refine=False)
+
+        show_plots = False
+
+        t = p.get_val('traj.phase.timeseries.time')
+
+        rho = p.get_val('traj.phase.timeseries.states:x0')
+        theta = p.get_val('traj.phase.timeseries.states:x1')
+        phi = p.get_val('traj.phase.timeseries.states:x2')
+
+        u0 = p.get_val('traj.phase.timeseries.controls:u0')
+        u1 = p.get_val('traj.phase.timeseries.controls:u1')
+        u2 = p.get_val('traj.phase.timeseries.controls:u2')
+
+        exp_out = p.model.traj.simulate()
+
+        t_exp = p.get_val('traj.phase.timeseries.time')
```

Comment:
_exp variables should be pulling data from the explicit output problem instance returned by simulate:

`t_exp = exp_out.get_val('traj.phase.timeseries.time')`